src/views/v2/list/event/date.php

```diff
@@ -13,11 +13,16 @@
  *
  */
 
-$event    = $this->get( 'event' );
-$event_id = $event->ID;
+$event       = $this->get( 'event' );
+$event_id    = $event->ID;
+$is_featured = tribe( 'tec.featured_events' )->is_featured( $event_id );
+
 ?>
-<div class="tribe-events-calendar-list__event--datetime">
-	<time datetime="1970-01-01T00:00:00+00:00">
+<div class="tribe-events-calendar-list__event-datetime-wrapper">
+	<time class="tribe-events-calendar-list__event-datetime tribe-common-b2" datetime="1970-01-01T00:00:00+00:00">
 		<?php echo tribe_events_event_schedule_details( $event ); ?>
 	</time>
-</div>
\ No newline at end of file
+	<?php if ( $is_featured ) : ?>
+		<span class="tribe-events-calendar-list__event-datetime-featured-icon"></span>
```

Comment:
If this span (guessing as an icon) is the only thing that denotes an event as being featured, it would be helpful to add a `title` & `aria-label` to that effect ;)